internal/datacoord/compaction_trigger.go

```diff
@@ -449,6 +449,11 @@ func (t *compactionTrigger) generatePlans(segments []*SegmentInfo, force bool, c
 	// TODO, currently we lack of the measurement of data distribution, there should be another compaction help on redistributing segment based on scalar/vector field distribution
 	for _, segment := range segments {
 		segment := segment.ShadowClone()
+		// by-pass stale segments
+		if !force && t.isStaleSegment(segment) {
+			log.Debug("generate plans skip stale segment", zap.Int64("segmentID", segment.GetID()), zap.Time("lastFlushTime", segment.lastFlushTime))
```

Comment:
the log level is correct?

Reply:
This log could be very frequent in some case, so put it in debug level. Which level you have in mind? @jiaoew1991